xml/tuning-ceph-tuning.xml

```diff
@@ -0,0 +1,724 @@
+           "req_setxattr_latency": {},
+           "req_symlink_latency": {},
+           "req_unlink_latency": {},
+       }
+</screen>
+     <para>
+       Tuning the metadata server cache allows for more metadata operations
+       to come from RAM, resulting in improved performance. The example
+       below sets the cache to 16&nbsp;GB.
+     </para>
+<screen>
+mds_cache_memory_limit=17179869184
+</screen>
+   </sect2>
+   <sect2>
+     <title>CephFS - Client</title>
+     <para>
+       From the client side, there are a number of performance affecting
+       mount options that can be employed. It is important to understand
+       the potential impact on the applications being utilized before
+       employing these options.</para>
+      <para>
+       The following mount options may be adjusted to improve
+       performance, but we recommend that their impact is clearly
+       understood prior to implementation in a production environment.
+     </para>
+     <variablelist>
+       <varlistentry>
+         <term>noacl</term>
+         <listitem>
+           <para>
+             Setting this mount option disables POSIX Access Control Lists
+             for the CephFS mount, lowering potential metadata overhead.
+           </para>
+         </listitem>
+       </varlistentry>
+       <varlistentry>
+         <term>noatime</term>
+         <listitem>
+           <para>
+             This option prevents the access time metadata for files
+             from being updated.
+           </para>
+         </listitem>
+       </varlistentry>
+       <varlistentry>
+         <term>nodiratime</term>
+         <listitem>
+           <para>
+             Setting this option prevents the metadata for access time of
+             a directory from being updated.
+           </para>
+         </listitem>
+       </varlistentry>
+       <varlistentry>
+         <term>nocrc</term>
+         <listitem>
+           <para>
+             This disables CephFS CRCs, thus relying on TCP Checksums for
+             the correctness of the data to be verified.
+           </para>
+         </listitem>
+       </varlistentry>
+       <varlistentry>
+         <term>rasize</term>
+         <listitem>
+           <para>
+             Setting a larger read-ahead for the mount may increase
+             performance for large, sequential operations. Default is 8&nbsp;MiB.
+           </para>
+         </listitem>
+       </varlistentry>
+     </variablelist>
+   </sect2>
+ </sect1>
+ <sect1 xml:id="tuning-rgw">
+   <title>RGW</title>
+   <para>
+     There are a large number of tunables for the Rados GateWay (RGW).
+     These may be specific to the types of workloads being handled
+     by the gateway and it may make sense to have different gateways
+     handling distictly different workloads.
+   </para>
+   <sect2>
+     <title>Sharding</title>
+     <para>
+       The ideal situation is to understand an idea of how many total
```

Comment:
"understand an idea how many"?
I would think "understand how many" or failing that, "get an idea how many"...?